### DISCUSSION TITLE
API partial response

### DISCUSSION THREAD

**Discussion by jiblett1000** [mastodon/mastodon#17324] [Q&A] [answered]
Hey ya'll,

Just wondering if it's possible to get a partial response from the Mastodon API. Particularly, I am accessing this endpoint:

`https://mastodon.social/api/v1/accounts/:id/statuses`

I would like just the content, created_at and media_attachments fields from each status object.

**Comment by Gargron** [comment, marked as answer]
No, we do not currently support customizing fields returned in response.

**Reply by jiblett1000** [reply]
Ah, got it. I'll cease banging my head against the wall then. Thanks for the reply :)
